app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
```

Comment:
same loop used in line 11-14, 21-24, 33-36.  Its better to move it to different function and pass a param that is changing. Also its better not using _ as a variable if you are using it in the loop later